### PR TITLE
fix((un)install): write error messages to `stderr`

### PR DESCRIPTION
**Pull request by wizard-28**
## Purpose

Update `fancy_message` functions in `install.sh` and `uninstall.sh` to write error messages to `stderr`

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

**Comment by Henryws** [comment]
Fix the title

**Review by D-Brox — APPROVED** [review]

**Comment by wizard-28** [comment]
> Fix the title

It's editing both the files so the scope is `(un)install` idk

**Comment by Henryws** [comment]
It should be `fix((un)install): write error messages to `stderr``

**Review by Henryws — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -44,10 +44,10 @@ function fancy_message() {
 	local MESSAGE="${2}"
 
 	case ${MESSAGE_TYPE} in
-		info) echo -e "[${GREEN}+${NC}] INFO: ${MESSAGE}" ;;
-		warn) echo -e "[${YELLOW}*${NC}] WARNING: ${MESSAGE}" ;;
-		error) echo -e "[${RED}!${NC}] ERROR: ${MESSAGE}" ;;
-		*) echo -e "[?] UNKNOWN: ${MESSAGE}" ;;
+		info) echo -e "[${BGreen}+${NC}] INFO: ${MESSAGE}" ;;
```
Comment (D-Brox) [inline install.sh:47]: Fix variable names

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (wizard-28) [reply install.sh:47]: We use bolded variants in `pacstall`, so I copied that over to update it here, for consistency.

https://github.com/pacstall/pacstall/blob/5ada91da49a5bfec379041cf8bb8ca7eb834757b/pacstall#L122-L127

Comment (D-Brox) [reply install.sh:47]: Yes, but those aren't the names defined in this script:
image

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: install.sh [hunk 2]
```diff
@@ -23,12 +23,20 @@
...
 
 # Colors
+BOLD=$(tput bold)
+NORMAL=$(tput sgr0)
 NC="\033[0m"
 
 RED='\033[0;31m'
-YELLOW='\033[1;33m'
 GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```
Comment (D-Brox) [inline install.sh:32]:
```suggestion
```

File: install.sh [hunk 3]
```diff
@@ -23,12 +23,20 @@
...
+
+BRed='\033[1;31m'
+BGreen='\033[1;32m'
+BYellow='\033[1;33m'
+
 BIGreen='\033[1;92m'
+BIRed='\033[1;91m'
```
Comment (D-Brox) [inline install.sh:39]:
```suggestion
```
Also, replace `$BIGreen` with `$BGreen` in the rest of the code.

**Review by D-Brox — CHANGES_REQUESTED** [review]
``

**Pull request merged**